connection/src/thick.rs

```diff
@@ -435,6 +436,50 @@ impl<CP: CredentialsProvider> UserTxConnection for ThickClient<CP> {
             ))
         }
     }
+
+    fn propose_tx_v2(&mut self, tx: &Tx, fee_map: &FeeMap) -> Result<TxOkData> {
+        trace_time!(self.logger, "ThickClient::propose_tx_v2");
+
+        if !self.is_attested() {
+            let _verification_report = self.attest()?;
+        }
+
+        let enclave_connection = self
+            .enclave_connection
+            .as_mut()
+            .expect("no enclave_connection even though attest succeeded");
+
+        let req = ClientProposeTxRequestV2 {
+            tx: tx.clone(),
+            fee_map_digest: fee_map.canonical_digest().to_vec(),
+        };
+
+        let mut msg = Message::new();
+        msg.set_channel_id(Vec::from(enclave_connection.binding()));
+
+        // Don't leave the plaintext serialization floating around
+        let tx_plaintext = SecretVec::new(encode(&req));
```

Comment:
Yes, we are not zeroizing everything we should be.
Historically there were problems because Isis' upstream crate did not derive Zeroize on `RistrettoPoint`.

We should make a ticket to fix all the zeroize issues, but this is orthogonal to this PR, and we should not let perfect be the enemy of good.